openvino_tensorflow/backend.cc

```diff
@@ -41,17 +44,34 @@ Backend::Backend(const string& config) {
     ss << "Device '" << config << "' not found.";
     throw runtime_error(ss.str());
   }
+
+  if (device == "GPU" && prec != "" && prec != "FP16") {
+    stringstream ss;
+    if (prec == "FP32") {
+      ss << "'GPU_FP32' is not a supported device name."
+         << " Please use 'GPU' device name for FP32 precision.";
+      throw runtime_error(ss.str());
+    } else {
+      ss << "The precision '" << prec << "' is not supported on 'GPU'.";
+      throw runtime_error(ss.str());
+    }
+  } else if (device != "GPU" && prec != "") {
+    stringstream ss;
+    ss << "Device '" << device << "' does not support custom precisions.";
+    throw runtime_error(ss.str());
+  }
+
   m_device_type = config;
   if (config.find("MYRIAD") != std::string::npos) {
     m_device = "MYRIAD";
```

Comment:
Looks like this condition is too handle when multiple myriad devices are connected. Is it already tested?

Reply:
Right but this is implemented prior to this branch. This should have been tested earlier already. @suryasidd can you confirm?

Reply:
The MYRIAD condition is so that we can accept "MYRIAD" and also the entire device name as well if multiple myriads are connected. Yeah this was already present in our code right and it was validated.